powershell/VstsTaskSdk/ToolFunctions.ps1

```diff
@@ -144,14 +146,14 @@ function Merge-Pipelines {
     param(
         [Parameter(Mandatory = $true)]
         [scriptblock]$ScriptBlock,
-        [switch]$DoNotSetErrorActionPreference)
+        [switch]$NoClosure)
```

Comment:
I renamed this switch for clarity and moved setting error preference out of this function.